content/intermediate/241_pixie/using_pixie/mysql_data.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Inspect the MySQL Request"
+date: 2021-5-01T09:00:00-00:00
+weight: 24
+draft: false
+---
+
+From the demo app’s YAML file, we know that the catalog service talks to a MySQL database. Let’s inspect the `catalog` service's mysql requests to see if we can get more information about the type of database connection error.
+
+Select the `px/mysql_data` script from the script drop-down menu. This script shows all of the mysql requests Pixie has traced in the cluster. Let’s filter these requests.
+
+Modify the script’s `start_time` to `-30m` or any window that will include when you [triggered the bug](/intermediate/241_pixie/prereqs/#trigger-the-microservices-application-bug) in the Sock Shop app.
+
+Open the script editor using `ctrl+e` (Windows, Linux) or `cmd+e` (Mac).
+
+On line 34, add the following line to filter the mysql requests to just those with errors:
```

Comment:
I copy/pasted without regard to indentation, and saw an error. I think we should mention that the new lines should be spaced to match the existing spacing.